docs/tutorial/04_integrating_magma_access_gateway_with_magma_orchestrator.md

```diff
@@ -0,0 +1,90 @@
+# 4. Integrating Magma Access Gateway with Magma Orchestrator 
+
+## Integrate Magma Access Gateway with Magma Orchestrator
+
+Offer an application endpoint from Orchestrator:
+
+```console
+juju offer orc8r.orc8r-nginx:orchestrator
+juju consume orc8r.orc8r-nginx
+```
+
+Relate Magma Access Gateway with Orchestrator:
+
+```console
+juju relate orc8r-nginx:orchestrator magma-access-gateway-operator
+```
+
+Wait for the application to go back to `Active-Idle`:
+
+```console
+ubuntu@host:~$ juju status
+Model  Controller     Cloud/Region   Version  SLA          Timestamp
+edge   aws-us-east-2  aws/us-east-2  2.9.42   unsupported  16:09:01-05:00
+
+App                            Version  Status  Scale  Charm                          Channel  Rev  Exposed  Message
+magma-access-gateway-operator           active      1  magma-access-gateway-operator  stable    29  no       
+
+Unit                              Workload  Agent  Machine  Public address  Ports  Message
+magma-access-gateway-operator/0*  active    idle   0        18.189.227.182         
+
+Machine  State    Address         Inst id                Series  AZ  Message
+0        started  18.189.227.182  manual:18.189.227.182  focal       Manually provisioned machine
+```
+
+Fetch the Access Gateway's `Hardware ID` and `Challenge Key` and note those values:
+
+```console
+juju run-action magma-access-gateway-operator/0 get-access-gateway-secrets --wait
+```
```

Comment:
How about adding an example of the output?

Reply:
Done